apps/src/aichat/views/AichatView.tsx

```diff
@@ -60,6 +61,7 @@ const AichatView: React.FunctionComponent = () => {
         studentAiCustomizations,
       })
     );
+    dispatch(setNewChatSession());
```

Comment:
Is this the right place to put something that I want to happen when the level changes?

Reply:
Ok so in combination with your comment [here](https://github.com/code-dot-org/code-dot-org/pull/58103/files/1847f2eb2aedfb5a941d3ef601d9817206feff5d#r1575161767), I 

a) made it so any clearing of messages also clears the session (good call!), and 
b) moved the message clearing effect into `AichatView` from `ChatWorkspace`.

My thinking is that the chat history, since we're treating it as something that initiates a new session, it feels like something that should be managed at the top level, rather than in a more specific component like `ChatWorkspace`. LMK what you think!

Reply:
Nice!  I agree this it makes more sense to move to `AichatView`.  Looks great - thanks for the refactor.